src/views/search/search.jsx

```diff
@@ -96,6 +95,31 @@ var Search = injectIntl(React.createClass({
         }
         return allTab;
     },
+    getProjectBox: function () {
+        var results = <Grid items={this.state.loaded}
+                            itemType={this.props.tab}
+                            cards={true}
+                            showAvatar={true}
+                            showLoves={false}
+                            showFavorites={false}
+                            showViews={false} />;
+        var searchAction;
+        if (this.props.searchTerm === '') {
```

Comment:
We actually, if you look on production, will show results for a general query when the search term is empty. It appears that there is a separate bug right now for that where you need to click `Load More` to display the first set of results though. I'd recommend removing this check, and I can file a separate bug for the empty search issue.